exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,31 @@
+# Description
+
+Today, most of the world use Arabic numbers.
+But if you go back two thousand years, a large proportion of Europe were using Roman numerals instead.
+
+Roman numerals used letters from the Latin alphabet (I, V, X, L, C, D, M) to represent numbers, and rather than having digits of 0 to 10 like modern Arabic numbers, they had numerals representing numbers starting with 1 and 5.
```

Comment:
```suggestion
Roman numerals used letters from the Latin alphabet (I, V, X, L, C, D, M) to represent numbers. 
Rather than having digits of 0 to 10, they had numerals representing numbers starting with 1 and 5.
```